AzureSQLVM/e2e-ag-setup/README.md

```diff
@@ -0,0 +1,59 @@
+# End-to-End provision of Multi Subnet Availability Group for SQL Servers running on Azure Virtual Machines. 
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.sqlvirtualmachine%2Fe2e-sql-vm-ag-setup%2Fazuredeploy.json)  
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.sqlvirtualmachine%2Fe2e-sql-vm-ag-setup%2Fazuredeploy.json)
+
+`Tags: Azure, SQL, VirtualMachine, AlwaysON, Listener`
+
+## Solution overview and deployed resources
+
+This is an overview of the solution
+1. Creates availability set
+2. Creates multiple SQL VMs in availability set, each vm in a different subnet (Maximum number of VMs for this solution is 9, we recommend VM count > 2)
+3. Join SQL VMs to the domain 
+4. Creates Storage account (if it doesn't exist already) which will act as Witness for Failover Cluster
+5. Creates Windows server Failover cluster
+6. Runs necessary checks such as TEST Cluster to ensure the health of creation of cluster 
+7. Creates Availability Group 
+8. Creates Availability Group Listener
+
+Best practices that were considered for this solution:
+1. This solution deploys SQL Server VMs to multiple subnets,thereby avoiding the dependency on an Azure Load Balancer or a distributed network name (DNN) to route traffic to your HADR solution.
+2. Use a single NIC per cluster node.
+
```

Comment:
Sorry I was thinking that all of the best practices were implemented so I just wanted to include the link. I think it's better to just remove this section and not say anything about best practices until we have them all implemented.

Reply:
Ok, I will remove this section